workflow/rocoto/gefs_tasks.py

```diff
@@ -236,10 +236,9 @@ def _atmosoceaniceprod(self, component: str):
         if component in ['ocean']:
             dep_dict = {'type': 'data', 'data': data, 'age': 120}
             deps.append(rocoto.add_dependency(dep_dict))
-            command = f"{self.HOMEgfs}/ush/check_netcdf.sh {history_path}/{history_file_tmpl}"
-            dep_dict = {'type': 'sh', 'command': command}
+            dep_dict = {'type': 'task', 'name': f'{self.cdump}fcst'}
```

Comment:
There is no forecast task named "gefsfcst". For GEFS, there is a metatask called "fcst_ens" with tasks called "fcst_mem#member#" (perturbed members), and there is a task called "fcst_mem000" (control member).

Reply:
I believe this needs to be changed to `dep_dict = {'type': 'task', 'name': f'fcst_mem#member#'}`, otherwise the ocean_prod task for the final lead hour will stall.

Reply:
you are absolutely correct @EricSinsky-NOAA 
The differences between GFS and GEFS had me confused.
I have corrected the error. Thanks!

Reply:
Thanks for making this change.